### DISCUSSION TITLE
[SOLVED] Add support for tap gesture

### DISCUSSION THREAD

**Discussion by chandra-arifin** [CommunityToolkit/Maui.Markup#95] [New Feature Discussions]
https://youtu.be/ddmZ6k1GIkM

Is the markup will add support for tap gesture like this video?

And swipe right also not working.

If the tap gesture on parent control (label on parent control) will work fine. But when inside the item template, not working.

(as you see on video, the view model is link to ancestor type)

Maybe support for binding context on items template (collection view item template etc) with type ancestor like video?

**Comment by chandra-arifin** [comment]
i solved this by use the extension:

.BindTapGesture(nameof(vm.ShowItemCommand))

how can i closed this discussion?

**Reply by pictos** [reply]
We can't close the discussion. No worries. Glad that you've managed to solve it

**Reply by chandra-arifin** [reply]
just happy can use this maui markup extension. very helpful.  can i keep asking for new features? 
;p

**Reply by pictos** [reply]
@chandra-arifin absolutely! All new ideas are very welcome!

**Reply by chandra-arifin** [reply]
thanks. will try to keep improving this library, so much useful. still much bugs when try it, but so fun to me. thanks pictos

**Reply by pictos** [reply]
@chandra-arifin for bugs, please open issues following our template we will be more than happy to fix them.

**Comment by CliffAgius** [comment]
Closed and answered
